public/javascripts/Gallery/src/cards/CardContainer.js

```diff
@@ -47,6 +47,8 @@ function CardContainer(uiCardContainer) {
 
     let pagewidth;
 
+    let modal;
+
```

Comment:
Personal preference: I'd rather not have all this whitespace between the variables you're declaring unless there are obvious groupings to do.